src/types/expansion.ts

```diff
@@ -11,11 +12,30 @@ export type ExpansionRule = {
   id: number;
   name: string;
   owner: string;
+  tags: { tag: Tag }[];
   updated_at: string;
   updated_by: string;
 };
 
-export type ExpansionRuleInsertInput = Omit<ExpansionRule, 'created_at' | 'id' | 'updated_at' | 'updated_by' | 'owner'>;
+export type ExpansionRuleSimpleTags = {
```

Comment:
Change to `export type ExpansionRuleSimpleTags = Omit<ExpansionRule, 'tags'> & { tags: { tag_id: number }[]; }`

Reply:
And change to `ExpansionRuleSlim`